src/Microsoft.Azure.ServiceBus/Management/QueueRuntimeInfoExtensions.cs

```diff
@@ -0,0 +1,115 @@
+﻿namespace Microsoft.Azure.ServiceBus.Management
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Xml.Linq;
+
+    internal static class QueueRuntimeInfoExtensions
+    {
+        public static QueueRuntimeInfo ParseFromContent(string xml)
+        {
+            var xDoc = XElement.Parse(xml);
+            if (!xDoc.IsEmpty)
+            {
+                if (xDoc.Name.LocalName == "entry")
+                {
+                    return ParseFromEntryElement(xDoc);
+                }
+            }
+
+            throw new MessagingEntityNotFoundException("Queue was not found");
+        }
+
+        // TODO: is this used?
+        static IList<QueueRuntimeInfo> ParseCollectionFromContent(string xml)
```

Comment:
Nope. Not used. We should remove that given that we don't support getting multiple runtime infos at same time